### DISCUSSION TITLE
application top is cutted

### DISCUSSION THREAD

**Discussion by shaymolho** [airsdk/Adobe-Runtime-Support#1931] [Q&A] [answered]
![top-bar-cutted](https://user-images.githubusercontent.com/6802402/169851210-4586865b-5b4e-4dce-a30c-49e0debd685e.jpg)
This happens in android after starting to use apm.

@marchbold any idea?

**Comment by marchbold** [comment]
Have you compared the manifest that apm produces with the one you had previously?

**Reply by shaymolho** [reply]
[app-descriptor.zip](https://github.com/airsdk/Adobe-Runtime-Support/files/8760588/app-descriptor.zip)
Attached are both the previous and the new

**Reply by ajwfrost** [reply]
Wondering if there's an impact with the 'notch support' metadata - does the phone you're using have a notch in it? The other things (aspect ratio, full screen) are the same (although I'm not 100% sure that the aspect ratio request is in the normal place?)

Can you confirm that the same AIR SDK build is being used for the before/after cases? and which version that is? Plus the Android OS version (and maybe device model) would be handy too.

thanks

**Reply by shaymolho** [reply]
The devices which it happened are
HUAWEI HMA-L29 (Huawei Mate 20)
HUAWEI MAR-LX1A (Huawei P30 Lite)

I'm not sure if the same air sdk, but the new one that is having this problem is 821

**Reply by ajwfrost** [reply]
Okay thanks -> there was a change regarding max_aspect that we put into build 856, so it won't be that if you're using 821..

The suggestion earlier I think was to compare the actual (generated) AndroidManifest.xml files between the two builds, are you able to do that? If you have the APK files you can open them up in Android Studio to view them? It may be easier to find the issue when looking at those rather than just the AIR application descriptor files.

thanks

**Reply by marchbold** [reply]
How are you setting the display mode? i.e. fullscreen / immersive etc? 

The notch support flag definitely seems to be the major difference, have you tried removing it?

**Reply by shaymolho** [reply]
Yes it was the notch setting, I've removed it
thanks

**Reply by marchbold** [reply]
Were you doing anything to the display mode? Or just relying on AIR with the fullscreen flag?

Did you test on any devices other than the huawei ones above?

**Comment by shaymolho** [comment]
[old-manifest.txt](https://github.com/airsdk/Adobe-Runtime-Support/files/8761600/old-manifest.txt)
[new-manifest.txt](https://github.com/airsdk/Adobe-Runtime-Support/files/8761602/new-manifest.txt)

attached both old and new manifest.xml
thanks for the quick replies

**Comment by shaymolho** [comment, marked as answer]
The solution was to remove     
`<meta-data android:name="android.notch_support" android:value="true"/>`
from app-descriptor
